jetstream/logging/__init__.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from jetstream.logging.bigquery_log_handler import BigQueryLogHandler
+
+
+def setup_logger(
+    log_project_id, log_dataset_id, log_table_id, log_to_bigquery, client=None, capacity=50
+):
+    logging.basicConfig(
+        level=logging.INFO,
+        format="%(levelname)s:%(asctime)s:%(name)s:%(message)s",
+    )
+    logger = logging.getLogger()
+
+    if log_to_bigquery:
+        bigquery_handler = BigQueryLogHandler(
+            log_project_id, log_dataset_id, log_table_id, client, capacity
+        )
+        bigquery_handler.setLevel(logging.WARNING)
+        logger.addHandler(bigquery_handler)
```

Comment:
This is small enough to stuff in cli.py and that's the only place we should call it from, so we might as well put it there.